tests/cypress/integration/case_6_image_scale_fit.js

```diff
@@ -32,17 +34,24 @@ context('Check if the image is scaled and then fitted', () => {
     describe(`Testing "${labelName}"`, () => {
         it('Scale image', () => {
             cy.get('#cvat_canvas_background')
-                .should('have.attr', 'style').and('contain', 'scale(1.05)')
+            .should('have.attr', 'style')
+            .then($scale => {
```

Comment:
I suppose it isn't scale, it's ``styles``

Reply:
This is getting all the parameters of the style attribute.
For example:
```
top: 20px; left: 73px; width: 800px; height: 800px; filter: brightness(1) contrast(1) saturate(1); transform: scale(1.05) rotate(0deg);
```

 Then use the regular expression to get the scale value.
For example:
```
1.05
```

Reply:
Yes, I mean that the parameter of then() - ``scale`` is not actual scale, it's all the styles where you find scale with regex.

Reply:
The issue about naming

Reply:
I understood. Thanks. Corrected.